compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/window/DesktopPopupTest.kt

```diff
@@ -181,14 +189,19 @@ class DesktopPopupTest {
             }
         }
 
-        rule.sendBackEvent()
+        navEventInput.backCompleted()
         assertThat(onDismissRequestCallCount).isEqualTo(1)
     }
 
     @Test
     fun doNotDismissPopupByBackEventWithoutBackPressProperty() {
         var onDismissRequestCallCount = 0
+        val navEventInput = DirectNavigationEventInput()
         rule.setContent {
+            val owner = findDefaultNavigationEventDispatcherOwner()
+            LaunchedEffect(Unit) {
```

Comment:
Probably it should be
```suggestion
            LaunchedEffect(owner) {
```